v4/source/definitions.yaml

```diff
@@ -2202,6 +2202,23 @@ definitions:
         type: string
         description: The id of the default channel user role for this scheme.
 
+  ServiceTerms:
```

Comment:
Propose the feature is referred to as "Terms Of Service" in all user-facing texts

Reply:
This can be `TermsOfService` for instance when no spaces are permitted.

Reply:
Yes, that is correct. Make the changes.